include/SDL/SDL_stdinc.h

```diff
@@ -83,10 +83,9 @@ SDL_COMPILE_TIME_ASSERT(enum, sizeof(SDL_DUMMY_ENUM) == sizeof(int));
 #ifdef HAVE_STDIO_H
 #include <stdio.h>
 #endif
-#ifdef HAVE_STDLIB_H
+#if defined(HAVE_STDLIB_H)
```

Comment:
Aren't these two equivalent?

Reply:
Yes. The important part is the use of an `#elif` instead of a second, separate `#if`.